Resources/Prototypes/Actions/psionics.yml

```
@@ -39,9 +39,9 @@
   components:
     - type: EntityTargetAction
       icon: Interface/VerbIcons/mind_swap.png
-      useDelay: 120
+      useDelay: 10
```

```suggestion
      useDelay: 120
```
Maintainers are not allowing this, keep delay at 120.
Note: if you wish to change the revert, check ActionMindSwapReturn.

---

Alright. I merely was trying to give the mindswap power a delay that enable it to function relatively well despite it having the do over bar and ability to fail at a moments notice